specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/expressRoutePort.json

```diff
@@ -884,6 +1130,13 @@
           },
           "description": "Reference the ExpressRoute circuit(s) that are provisioned on this ExpressRoutePort resource."
         },
+        "authorizations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExpressRoutePortAuthorization"
```

Comment:
Parents should not contain child resources in their body.  This breaks RBAC and makes policy, etc... harder for users.  Either it is a child resource or it is a property in the port, not both.

Reply:
I see our other resources having child resources as a property - is this something we started to mandate recently? - either ways resolved it.

Reply:
Its a bad pattern that we are trying to get rid of